addons/test_mail/tests/test_mail_composer.py

```diff
@@ -453,6 +453,37 @@ def test_mail_composer_rights_attachments(self):
             sorted(self.test_record.message_ids[0].attachment_ids.mapped('name')),
             sorted(template_1_attachment_name))
 
+    @users('employee')
+    def test_mail_composer_template_switching(self):
```

Comment:
There is a ``TestComposerForm`` class in this file for tests that more specifically use the Form tool to check what happens to the composer values ... Could be moved in that section :) (and there are a lot of additional tests in next version in that class so it makes sense to add this one there I think)

Reply:
sure thing, moving to TestComposerForm